lib/quicly.c

```diff
@@ -2795,6 +2795,7 @@ static int commit_send_packet(quicly_conn_t *conn, quicly_send_context_t *s, enu
     if (quicly_sentmap_is_open(&conn->egress.loss.sentmap))
         quicly_sentmap_commit(&conn->egress.loss.sentmap, (uint16_t)packet_bytes_in_flight);
 
+    conn->egress.cc.impl->cc_on_sent(&conn->egress.cc, &conn->egress.loss, (uint32_t)packet_bytes_in_flight, conn->stash.now);
```

Comment:
After going through the `quicly_send` and `do_send` functions, I felt like this is the best place to put the callback. But I'm not that familiar with quicly's architecture, so suggestions for better locations are welcome.

Reply:
I think this is fine.